src/main/java/de/tum/cit/aet/artemis/core/domain/LLMServiceType.java

```diff
@@ -0,0 +1,6 @@
+package de.tum.cit.aet.artemis.core.domain;
+
+public enum LLMServiceType {
+    ATHENA, IRIS_CODE_FEEDBACK, IRIS_CHAT_COURSE_MESSAGE, IRIS_CHAT_EXERCISE_MESSAGE, IRIS_INTERACTION_SUGGESTION, IRIS_CHAT_LECTURE_MESSAGE, IRIS_COMPETENCY_GENERATION,
```

Comment:
For Athena we can distinguish between  `ATHENA_PRELIMINARY_FEEDBACK` and `ATHENA_FEEDBACK_SUGGESTION`.